src/components/dashboard/OutOfGasError.js

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import _get from 'lodash/get'
+import * as web3Utils from 'web3-utils'
+import normalize from '../../lib/utils/normalizeText'
+import GDStore from '../../lib/undux/GDStore'
+import goodWallet from '../../lib/wallet/GoodWallet'
+import { AwaitButton, Section, Wrapper } from '../common'
+import Separator from '../common/layout/Separator'
+import Oops from '../../assets/oops.svg'
+import logger from '../../lib/logger/pino-logger'
+
+const log = logger.child({ from: 'OutOfGasError' })
+
+const OutOfGasError = props => {
+  const MIN_BALANCE_VALUE = '100000'
+  const gdstore = GDStore.useStore()
+  const isValid = _get(props, 'screenProps.screenState.isValid', undefined)
+
+  const ERROR =
+    "In·order·for·transactions·to·go·through,\nyou·need·some·virtual·money·named·'gas'.\nDon't·worry,·we'll·take·care·off·you.\n"
+  const ERROR_BOLD = "We're giving it to you for FREE, FOREVER."
+  const TITLE = "Ooops,\nYou're out of gas..."
+
+  if (isValid) {
+    props.screenProps.pop({ isValid })
+  }
+
+  const [isLoading, setLoading] = useState(false)
+  const { balance } = gdstore.get('account')
+
+  const gotoDb = () => {
+    props.screenProps.navigateTo('Home')
+  }
+
+  useEffect(() => {
+    callTopWallet()
+  }, [])
+
+  useEffect(() => {
+    if (parseInt(balance) >= Number(MIN_BALANCE_VALUE)) {
+      gotoDb()
+    }
+  }, [balance])
+
+  const callTopWallet = async () => {
+    setLoading(true)
+
+    const isOk = await goodWallet.verifyHasGas(web3Utils.toWei(MIN_BALANCE_VALUE, 'gwei'))
+
+    setLoading(false)
+
+    if (isOk) {
```

Comment:
if not ok we change button to "contact support" (pressing on it leads to our support form, see sidemenu)
also if not ok show error text "Something went wrong try again later"

Reply:
done

Reply:
I dont see the "something went wrong..."

Reply:
https://github.com/GoodDollar/GoodDAPP/pull/443/commits/c77214101917f7d1d863e4ff5f57336040b2b20a